### PR TITLE
Message for the next BWCloud shutdown

### PR DESCRIPTION
**Pull request by gmauro**

### CODE REVIEW

File: _data/notices.yml [hunk 1]
```diff
@@ -48,14 +48,18 @@
...
            
-# - title: Restricted compute capacity on the 30.01.2021 
-#   class: info # one of [success, info, warning, danger]
-#   site: freiburg # Your site
-#   messages:
-#       - message: |
-#           Due to upgrade work to our peta-storage we need to shutdown large parts of our compute infrastructure on the
-#           30.01.2021. We are hopeful to keep the webserver running and accept jobs and workflows, but there will be a delay
-#           in processing them.
+ - title: Limited computing capacity on next March 14th and 15th, 2021
```
Comment (beatrizserrano) [inline _data/notices.yml:51]:
```suggestion
 - title: Limited computing capacity on next April 14th and 15th, 2021
```

File: _data/notices.yml [hunk 2]
```diff
@@ -48,14 +48,18 @@
...
+   class: info # one of [success, info, warning, danger]
+   site: freiburg # Your site
+   messages:
+       - message: |
+           Due to upgrade work to the deNBI cloud infrastructure, we need to shut down mostly all of our computing resources next March 14th and 15th, 2021. 
+           The shut-down will start at <b>17:00 of March 14th</b> and, everything should return to normal operation by the end of the next day.
+           We will keep the webserver running, accepting jobs and workflows, but there will be a delay in processing them.
```
Comment (beatrizserrano) [inline _data/notices.yml:58]:
```suggestion
           We will keep the web server running, accepting jobs and workflows, but there will be a delay in processing them.
```

File: _data/notices.yml (hunk 1 above)
Comment (beatrizserrano) [reply _data/notices.yml:51]: time flies!

**Comment by gmauro** [comment]
Sankt @beatrizserrano

File: _data/notices.yml [hunk 3]
```diff
@@ -48,14 +48,18 @@
...
-#           Due to upgrade work to our peta-storage we need to shutdown large parts of our compute infrastructure on the
-#           30.01.2021. We are hopeful to keep the webserver running and accept jobs and workflows, but there will be a delay
-#           in processing them.
+ - title: Limited computing capacity on next April 14th and 15th, 2021
+   class: info # one of [success, info, warning, danger]
+   site: freiburg # Your site
+   messages:
+       - message: |
+           Due to upgrade work to the deNBI cloud infrastructure, we need to shut down most of our computing resources next April 14th and 15th, 2021. 
+           The shut-down will start at <b>17:00 of April 14th</b> and everything should return to normal operation by the end of the next day.
```
Comment (beatrizserrano) [inline _data/notices.yml:57]: is this 17 CEST? would be good to indicate I think

Comment (gmauro) [reply _data/notices.yml:57]: Yes, and please change it if any training need more time that day

Comment (beatrizserrano) [reply _data/notices.yml]: there's just one running on that week (in the US) and I already informed them about the shutdown. I will change CET to CEST and the rest looks good to me, thanks @gmauro!

**Review by beatrizserrano — APPROVED** [review]

File: _data/notices.yml (hunk 3 above)
Comment (gmauro) [reply _data/notices.yml]: I need some vacations :) Thank  you

**Pull request merged**